### PR TITLE
add example

### PR DESCRIPTION
**Pull request by isaac-looker**
Demo for https://github.com/Spelman-College/spelman-dashboard/pull/59

This is omitting `Tenured` keys, since they are a column dimension/attribute- that logic is TBD. 

To test this (output shown [here](https://gist.github.com/isaac-looker/0fc2d0a0e8035f6968f857fde0a85bc0) )

```
npx tsc example.ts --target esnext  --module commonjs && node example.js
```

### CODE REVIEW

File: frontend/src/queries/example.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import { DCIDS } from "./example_dcids"
+
+
+const Categories = {
+    'gender': new Set<string>([
+        'Male',
+        'Female',
+    ]),
+    'ethnicity': new Set<string>([
+        'HispanicOrLatino',
```
Comment (cernst122) [inline frontend/src/queries/example.ts:13]: Due to a DataCommons nuance, 'ethnicity' includes only 'HispanicOrLatino', and everything else is 'race'. Again on line 57

File: frontend/src/queries/example.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        'NonScienceAndEngineeringOccupation',
+    ]),
+}
+
+// Dimensions to Category mapping.
+const Dim2Cat = {}
+// console.log(Categories)
+
+Object.keys(Categories).forEach((key) => {
+    console.log(key)
```
Comment (cernst122) [inline frontend/src/queries/example.ts:41]: Consider removing log statement.

File: frontend/src/queries/example.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    const values = Categories[key]
+    values.forEach((value) => {
+        Dim2Cat[value] = key
+    })
+})
+// console.log(Dim2Cat)
+
+const Prefix = 'Count_Person_ScienceAndEngineeringRelatedMajor_EducationalAttainmentDoctorateDegree'
```
Comment (cernst122) [inline frontend/src/queries/example.ts:49]: I believe your dcid slicing logic will be able to handle this, but want to confirm we're on the same page that
the prefix won't be static & that the whole dcid will need to be generated in alphabetical order of its properties (not the values of the properties).

File: frontend/src/queries/example_dcids.ts
Comment (cernst122) [inline frontend/src/queries/example_dcids.ts:1]: FYI if you're working off the tenure 9-26 table, there was a slight change to the schema recently to include facultyType. Newly generated output is at https://docs.google.com/spreadsheets/d/1RkbT41iTtg2C-vLCKwNh0PzMecfIJIA5Mau9p6qxDoA/edit?resourcekey=0-nji8UXo8Wd0bJ_FUArJxXg#gid=1749978453. Table is now on its way to live DataCommons.

File: frontend/src/queries/example.ts (hunk 3 above)
Comment (isaac-looker) [reply frontend/src/queries/example.ts:49]: Are these not the DCIDs we're going to use for querying? If they are, they're not in alphabetical order.

File: frontend/src/queries/example_dcids.ts
Comment (isaac-looker) [reply frontend/src/queries/example_dcids.ts:1]: Thanks! These new keys are unpacking the pivot table which, was using Tenure, into rows. This means we won't need special case logic for the column view.

File: frontend/src/queries/example.ts (hunk 2 above)
Comment (isaac-looker) [reply frontend/src/queries/example.ts:41]: This PR is just a demo; I wasn't planning on merging. I was planning to write tests for querying confirmed datasets once we have a list of DCIDs that are ready for publishing.

File: frontend/src/queries/example.ts (hunk 1 above)
Comment (isaac-looker) [reply frontend/src/queries/example.ts:13]: My category names, 'gender', 'ethnicity', are my invention. I checked the keys and `HispanicOrLatino` appears to behave as if it's in the same category as other 'race` dimensions. Do we need to confirm that the `HispanicOrLatino` category/dimension is a synonym for the 'race' dimension before including it? I'm not seeing both 'race' and 'ethnicity' in any of the listed DCIDs.

File: frontend/src/queries/example.ts (hunk 3 above)
Comment (cernst122) [reply frontend/src/queries/example.ts:49]: Aligned offline, these are the real DCIDs and they're sorted based on the convention at https://docs.google.com/document/d/1YtmQu_gOXENt4W2lmlK-8Mtnw_XB3kZlLBjSN-gT0oE/preview#bookmark=kix.bdksa8hbv3si.

**Comment by isaac-looker** [comment]
Added tests in https://github.com/Spelman-College/spelman-dashboard/pull/62

**Pull request closed without merging**